Aspire Budgeting/PreviewProvider/MockProvider.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MockProvider.swift
+//  Aspire Budgeting
+//
+
+import Foundation
+
+struct MockProvider {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/845606672c945b4a26becac88e482112dc55b51f/Aspire Budgeting/PreviewProvider/MockProvider.swift#L8"></span>Types used for hosting only static members should be implemented as a caseless enum to avoid instantiation.<br />
<code>convenience_type</code> <code>MockProvider.swift:8</code></td>
    </tr>
  </tbody>
</table>